frontends/web/src/routes/account/send/send.tsx

```diff
@@ -765,12 +765,16 @@ class Send extends Component<Props, State> {
                                 <div className={style.confirmItem}>
                                     <label>{t('send.fee.label')}{feeTarget ? ' (' + t(`send.feeTarget.label.${feeTarget}`) + ')' : ''}</label>
                                     <p>
-                                        <span>{proposedFee && proposedFee.amount || 'N/A'} {proposedFee && proposedFee.unit || 'N/A'}</span>
-                                        {
-                                            proposedFee && proposedFee.conversions && (
-                                                <span> <span className="text-gray">/</span> {proposedFee.conversions[fiatUnit]} {fiatUnit}</span>
-                                            )
-                                        }
+                                        <span key="amount">{proposedFee && proposedFee.amount || 'N/A'} <small>{proposedFee && proposedFee.unit || 'N/A'}</small></span>
+                                        {proposedFee && proposedFee.conversions && (
+                                            <span key="conversation">
+                                                <span className="text-gray"> / </span>
+                                                {proposedFee.conversions[fiatUnit]} <small>{fiatUnit}</small>
+                                            </span>
+                                        )}
+                                        {feePerByte ? (
```

Comment:
you probably need to check that the fee target is set to custom as well?

otherwise, if you go to custom, set a fee per byte, then go back to a predefined target like 'low', then feePerByte might still be set to the previous value and shown in the dialog?

Reply:
Right, now it always resets the feePerByte to '' when changing the feetarget
https://github.com/digitalbitbox/bitbox-wallet-app/pull/995/commits/e5adf72002409307ce96ee18b0fb3e6e1009ab0b ack?

Reply:
ack. need to test again with a device though.

Reply:
I'm looking into passing the sat/vB info up to the parent component (sent.tsx) to display it in the confirm dialog…

Reply:
> I'm looking into passing the sat/vB info up to the parent component (sent.tsx) to display it in the confirm dialog…

dropped, for now it will only show if custom fee was selected